Karma/Views/Profile/Index.cshtml

```diff
@@ -3,6 +3,12 @@
 @{
     ViewData["Title"] = "Profile";
 }
+
+<head>
+    <link href="~/css/rating.css" type="text/css" rel="stylesheet" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Maybe it would be better if we had this specific `css` file locally?

Reply:
Are you talking about the font-awesome sheet?

Reply:
Yes.